### DISCUSSION TITLE
Xcode 16 Beta Package Naming Crash

### DISCUSSION THREAD

**Discussion by andykent** [groue/GRDB.swift#1581] [General]
I wasn't sure where to put this and I'm not sure any direct action is needed from this package yet but just wanted to flag for others that might hit this. 

There seems to be a bug in Xcode 16 betas and Xcode Cloud running on macOS 15 causing it to crash in `unregisterDependencyFileReferences(_:)` when resolving packages that contain a dot character in the package URL. 

See the thread here: https://developer.apple.com/forums/thread/758220

This would obviously impact `GRDB.swift` so hopefully it can be resolved on the Xcode side soon. 🤞

**Comment by groue** [comment]
Thanks for reporting this, @andykent.

I'm not going to change the URL of this repository because of a bug in Xcode beta, I'm sure you understand.

Please [file a feedback](https://feedbackassistant.apple.com). Filing bug reports is what we're supposed to do when we use beta software and find a bug. I hope it's clear the bug is not here.

**Reply by andykent** [reply]
oh yes, of course I am in no way expecting you to change the project name. 😄

Agreed it's not a GRDB issue hence I didn't file an issue and posted as a discussion instead, just thought it was worth mentioning here in case others hit it.

I have filed feedback with Apple already, I was hoping it would have been fixed in beta 4 but unfortunately it is not.

**Reply by groue** [reply]
Oh, you're right. Let's also reopen because I did not even intend to close the discussion.

Thank you for filing a feedback. Sometimes we indeed have to wait…

**Comment by andykent** [comment]
This seems to be fixed in Xcode 16 beta 5 🎉
